File: SDKNodeJs/models/UserInfosModel.js
module.exports = UserInfosModel;

function UserInfosModel() {
    this.PartenaireID;
    this.Mail;
    this.CodeUtilisateur;
    this.TypeAbonnement;
    this.DateExpiration ;
    this.DateSouscription;
    this.AccountExist;
}

UserInfosModel.prototype.CreateDummyModel = function () {
    this.Mail = "dummy@gmail.com";
    this.CodeUtilisateur = "dummy";
    this.AccountExist = true;
}

File: python_django/sdk_lpl/models/UserInfosModel.py
class UserInfoModel(object):
    PartenaireID = 0
    Mail = ""
    CodeUtilisateur = ""
    TypeAbonnement = ""
    DateExpiration = ""
    DateSouscription = ""
    AccountExist = False

    def __init__(self, **kwargs):
        self.__dict__.update(kwargs)

    def create_dummy_model(self):
        self.Mail = "dummy@gmail.com"
        self.CodeUtilisateur = "dummy1234"
        self.AccountExist = True
        self.PartenaireID = 0
